src/components/proposals/proposal-timeline.svelte

```diff
@@ -137,7 +137,7 @@
             {projectManagement.department.name}
 
             <span class="text-foreground/50"
-              >~ {(20 * +projectManagement.rate_value * +projectManagement.team_size) / 100}
+              >~ {(30 * +projectManagement.rate_value * +projectManagement.team_size) / 100}
```

Comment:
I'm not into the code but since the other changes you use a variable from utils shouldn't this number be one too?

Reply:
I opened the PR yesterday, noticed this, fixed it and forgot to push the changes to github. Thanks for noticing 👍